block-producer/src/store/mod.rs

```diff
@@ -93,18 +112,25 @@ impl ApplyBlock for DefaultStore {
 
 #[async_trait]
 impl Store for DefaultStore {
+    #[allow(clippy::blocks_in_conditions)] // Workaround of `instrument` issue
+    #[instrument(target = "miden-block-producer", skip_all, err)]
     async fn get_tx_inputs(
         &self,
         proven_tx: SharedProvenTx,
     ) -> Result<TxInputs, TxInputsError> {
-        let request = tonic::Request::new(GetTransactionInputsRequest {
+        let message = GetTransactionInputsRequest {
             account_id: Some(proven_tx.account_id().into()),
             nullifiers: proven_tx
                 .input_notes()
                 .iter()
                 .map(|nullifier| (*nullifier).into())
                 .collect(),
-        });
+        };
+
+        info!(target: COMPONENT, tx_id = %proven_tx.id().inner());
```

Comment:
Similar comment to the one before: let's use `to_hex()` here for transaction ID.